src/codeu/chat/util/Serializers.java

```diff
@@ -40,10 +40,9 @@ public Boolean read(InputStream in) throws IOException {
     @Override
     public void write(OutputStream out, Integer value) throws IOException {
 
-      out.write(0xFF & (value >> 24));
-      out.write(0xFF & (value >> 16));
-      out.write(0xFF & (value >> 8));
-      out.write(0xFF & (value >> 0));
+      for (int i = 24; i >= 0; i -= 8) {
+        out.write(0xFF & (value >>> i));
```

Comment:
Do you have unit tests for this. Please write a test for the original behavior and make sure the new behavior preserves it.